_editorial_policies/01_bylaws.md

```diff
@@ -144,3 +145,5 @@ In the case that all of the Section Lead Editors have potential conflicts, the M
 
    These bylaws may be amended by two-thirds of the vote of the entire
    Editorial Board.
+
+This version of the bylaws dated December 12, 2017.
```

Comment:
Perhaps this should be at the top of the doc?

Reply:
I'm OK either way, though this may make sense.

Reply:
Eh, I kind of like it at the bottom, since we we don't need the focus of new readers on that -- it's not going to be updated frequently (at least after the first year).   Maybe we'll see at the next revision if it should switch?